src/test/java/seedu/duke/commands/task/TaskListCommandTest.java

```diff
@@ -0,0 +1,35 @@
+package seedu.duke.commands.task;
+
+import org.junit.jupiter.api.Test;
+import seedu.duke.project.Project;
+import seedu.duke.project.ProjectList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class TaskListCommandTest {
+
+    @Test
+    void executeCommand() {
+        //Setup test project with existing tasks
+        ProjectList projects = new ProjectList();
+        projects.createProject("iPhone App");
+        projects.createProject("Android App");
+        int testIndex = 1;
```

Comment:
Would it be clearer if the name of the variable is changed to secondProjectIndex.

Reply:
Agree